linode/manager
# M3-5842: SMS Messaging Opt Out

## Description 📝

- Allows users to opt out of SMS messaging
- Relates to #8381 
- Essentially, a user is _opted in_ when they have a phone number, and _opted out_ when they don't have a phone number

## Preview 🔍

https://user-images.githubusercontent.com/6440455/171445887-36528660-66dc-456d-bc13-2668ed56317c.mov

## How to test 🧪

- Go to `http://localhost:3000/profile/auth`
- Turn on the MSW
- Try opting out of SMS messaging
  - The flow should be fully functional
